### DISCUSSION TITLE
1.17.1?

### DISCUSSION THREAD

**Discussion by gobboo** [fnuecke/oc2#154] [General]
Any support for 1.17.1, and if it's possible would it be easy to downport? Never touched modding but know how to program so I'd be happy to look into it but obviously no experience with modding, have Java experience thought just wondering if it'd be an easy task or not :)

**Comment by fnuecke** [comment]
Not planned, and if someone does it, they'd have to be the one to maintain it, too. Might not be the easiest pick for the first dive into modding; I believe in particular rendering changed quite a bit, so specifically the cables and projector would probably be the biggest pain to downport (if possible at all, for the latter).
